geometry_msgs/Pose robot_pose
string reason
---